docs/coverage.md

```diff
@@ -55,26 +55,78 @@ echo "coverage report at file://${destdir}/index.html"
 
 ```
 
-### Support for testing frameworks
+## Support for testing frameworks
 
 Coverage support has been only tested with [ScalaTest](http://www.scalatest.org/).
 
-### Working around missing lambda coverage with Scala 2.12+
+## JaCoCo
 
-The current Jacoco version in Bazel (0.8.3) has missing coverage for lambdas
-(including for comprehensions; see issue https://github.com/bazelbuild/rules_scala/issues/1056). Also, the support for
-filtering out code generated by the Scala compiler is quite reduced in Jacoco.
+`rules_scala` uses the [JaCoCo](https://www.jacoco.org/jacoco/) library imported
```

Comment:
Thank you for adding this!

Reply:
My pleasure! Thanks for taking such a thorough look over everything!